packages-native/actions/package.json

```diff
@@ -9,13 +9,24 @@
   "scripts": {
     "prebuild": "rm -rf ./dist/tsc",
     "build": "npm run build:tsc",
-    "build:tsc": "..\"/../node_modules/.bin/tsc\" --project src/tsconfig.json",
-    "postbuild": "..\"/../node_modules/.bin/ts-node\" --project ./scripts/tsconfig.json ./scripts/copyActionsToTestProject.ts",
+    "build:tsc": "tsc --project src/tsconfig.json",
+    "postbuild": "ts-node --project ./scripts/tsconfig.json ./scripts/copyActionsToTestProject.ts",
     "version": "npm run build",
-    "postversion": "..\"/../node_modules/.bin/ts-node\" ../../scripts/release/UpdateGithubRelease.ts --project ../../scripts/tsconfig.json"
+    "postversion": "ts-node ../../scripts/release/UpdateGithubRelease.ts --project ../../scripts/tsconfig.json"
+  },
+  "dependencies": {
```

Comment:
Same goes here, we should not have dependencies being shipped with the JS Actions as all those dependencies are already shipped with Studio Pro